src/Containers/PaletteForm/PaletteForm.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { addPalette } from '../../actions/actions';
+import './PaletteForm.scss';
+
+const PaletteForm = (props) => {
+
+  const [ project, selectProject ] = useState('');
+  const [ paletteName, setPalette ] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const paletteToAdd = {
+      name: paletteName,
+      project_id: project
+    }
+
+    props.palettes(paletteToAdd)
+    resetInputs();
+  }
+
+  const resetInputs = () => {
+    selectProject('');
+    setPalette('');
+  }
+
+  return(
+    <form>
+      <label for='select-project'>Select Project: </label>
+      <select
+        id='select-project'
+        className='project-selector'
+        value={project}
+        onChange={ e => selectProject(e.target.value) }
+      >
+        <option>One</option>
+        <option>Two</option>
+      </select>
+      <input
+        placeholder='Name Your Palette'
+        className='palette-name-input'
+        type='text'
+        maxLength='30'
+        value={paletteName}
+        onChange={ e => setPalette(e.target.value) }
+      />
+      <button className='select-palette-btn' type='button' onClick={ e => handleSubmit(e) }>Save Palette</button>
+    </form>
+  )
+}
+
+export const mapDispatchToProps = dispatch => ({
+  palettes: (palette) => dispatch(addPalette(palette))
```

Comment:
I think we should change the name of the this method to addPalette so that it matches the action and is a little more readable